exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/customize.ipynb

```diff
@@ -0,0 +1,300 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b61aa708-9383-4e5b-b072-b50492604f9c",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "### Open Secure Configuration Storage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "afd25eb3-a320-4375-8f5c-07ade762f28f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run ../utils/access_store_ui.ipynb\n",
+    "display(get_access_store_ui('../'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ef77c92-a796-4b02-9747-f711dca9a9b6",
+   "metadata": {},
+   "source": [
+    "### Instantiate SLCT Manager\n",
+    "\n",
+    "We create an instance of the SlctManager class from the notebook connector. SLCT Manager, which stands for \"Script-Languages-Container-Tools\" Manager. \n",
+    "This class has some utility functions which simplifies the use of the exaslct API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20c8c4cc-bc8a-45e8-b9df-b71021fd9476",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from exasol.nb_connector import slct_manager\n",
+    "slctmanager = slct_manager.SlctManager(ai_lab_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "264f7b58-093a-41db-8f09-c848896a1318",
+   "metadata": {},
+   "source": [
+    "### Import some utility functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "79e9843d-d436-4f97-b20e-56d2a42afdaa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run ./utils/file_system_ui.ipynb\n",
+    "%run ./utils/slc_ui.ipynb"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b632c4b5-3bb9-46c1-8c68-009102432a71",
+   "metadata": {},
+   "source": [
+    "## Customize\n",
+    "\n",
+    "First you need to define an alias for the new script-languages-container. The alias will be used to reference the container later from the UDF's.\n",
+    "\n",
+    "Note: In this tutorial the alias also will be used as part of the export file (tar.gz) and the uploaded container to the BucketFS. This allows you to create, upload and use different script-language-containers: one per alias."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a88f47d9-7337-45fb-b2d4-08b74bc95f5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "display(get_alias_ui(ai_lab_config, \"ai_lab_default_custom\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "410312a2-0976-423a-bdf5-2100a0085528",
+   "metadata": {},
+   "source": [
+    "### Flavor Customization Build Step\n",
+    "\n",
+    "`exasclt` consists of multiple build steps. By a build step here we mean a file structure which serves as an input for a certain stage of the building process of the script-languages-container. See [Advanced Topics](./advanced.ipynb) for more details.\n",
```

Comment:
```suggestion
    "`exasclt` performs multiple build steps. Each build step provides a file structure as input for the next stage of the build process. See [Advanced Topics](./advanced.ipynb) for more details.\n",
```

Reply:
Actually, here I find the original description more correct.